infrastructure/functions/main.tf

```diff
@@ -65,32 +59,34 @@ data "azurerm_storage_account_sas" "fap-backend-application-storage-account-sas"
   }
 }
 
-resource "azurerm_app_service_plan" "fap-backend-application-service-plan" {
-  name                = random_id.app_service_plan_name.id
-  location            = azurerm_resource_group.fap-function-store.location
-  resource_group_name = azurerm_resource_group.fap-function-store.name
+resource "azurerm_app_service_plan" "fap-backend-service-plan" {
+  name                = "fap-backend-service-plan"
+  location            = azurerm_resource_group.fap-backend-resource-group.location
+  resource_group_name = azurerm_resource_group.fap-backend-resource-group.name
   kind                = "functionapp"
+  tags                = var.azure_tags
   sku {
     tier = "Dynamic"
     size = "Y1"
   }
 }
 
-resource "azurerm_function_app" "fap-backend-application-function" {
-  name                       = var.azure_function_app_name
-  location                   = azurerm_resource_group.fap-function-store.location
-  resource_group_name        = azurerm_resource_group.fap-function-store.name
-  app_service_plan_id        = azurerm_app_service_plan.fap-backend-application-service-plan.id
-  storage_account_name       = azurerm_storage_account.fap-backend-application-storage-account.name
-  storage_account_access_key = azurerm_storage_account.fap-backend-application-storage-account.primary_access_key
+resource "azurerm_function_app" "fap-backend-function-app" {
+  name                       = "fap-backend-function-app"
+  location                   = azurerm_resource_group.fap-backend-resource-group.location
+  resource_group_name        = azurerm_resource_group.fap-backend-resource-group.name
+  app_service_plan_id        = azurerm_app_service_plan.fap-backend-service-plan.id
+  storage_account_name       = azurerm_storage_account.fap-backend-storage-account.name
+  storage_account_access_key = azurerm_storage_account.fap-backend-storage-account.primary_access_key
+  tags                       = var.azure_tags
```

Comment:
Die Tags als Variable zu hinterlegen ist sehr clever, sollte ich in meinem Code auch übernehmen